### PR TITLE
Rebuilt ti.sms iOS module into 64-bit

### PR DESCRIPTION
**Pull request by jithinv**
Rebuilt ti.sms iOS module into 64-bit

### CODE REVIEW

File: ios/documentation/changelog.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Change Log
 <pre>
+v1.1.0  Updated module to 64-bit architecture
```
Comment (jonalter) [inline ios/documentation/changelog.md]: Add the ticket to the changelog update.
Example:

```
v1.1.0 Updating module to support 64-bit [TIMOB-18092]
```

Make it look like the previous logs in that module's changlog where applicable

File: ios/manifest [hunk 2]
```diff
@@ -2,7 +2,9 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 1.0
+version: 1.1.0
+apiversion: 2
+architectures: armeabi armeabi-v7a x86
```
Comment (jonalter) [inline ios/manifest]: These are _android_ `architectures`

File: ios/titanium.xcconfig [hunk 3]
```diff
@@ -4,7 +4,7 @@
 // OF YOUR TITANIUM SDK YOU'RE BUILDING FOR
 //
 //
-TITANIUM_SDK_VERSION = 3.0.0.GA
+TITANIUM_SDK_VERSION = 3.4.1
```
Comment (jonalter) [inline ios/titanium.xcconfig]: Should be `3.4.1.GA`

**Comment by hansemannn** [comment]
Approved!

**Comment by hansemannn** [comment]
Approved!

**Pull request merged**